code/modules/projectiles/projectile/ego_bullets/aleph.dm

```diff
@@ -19,6 +19,6 @@
 	if(!isbot(H) && isliving(H))
 		H.visible_message("<span class='warning'>[target] is hit by [src], they seem to wither away!</span>")
 		for(var/i = 1 to 10)
-			addtimer(CALLBACK(H, /mob/living/proc/apply_damage, rand(4,6), BLACK_DAMAGE, null, H.run_armor_check(null, BLACK_DAMAGE)), 2 SECONDS * i)
+			addtimer(CALLBACK(H, /mob/living/proc/apply_damage, rand(6,8), BLACK_DAMAGE, null, H.run_armor_check(null, BLACK_DAMAGE)), 2 SECONDS * i)
```

Comment:
That's +20 damage.

Reply:
It's actually a slow firing weapon.
SOAS can do 116 damage per second at MINIMUM. 
Adoration with it's slow firing still loses out in dps to it.